### ISSUE TITLE
Moving GitHub issues that are used to explain process or structure of Marketing Team to Handbook

### ISSUE THREAD

**Issue by sereedmedia** [WordPress/Marketing-Team#180]
Per the title, I propose that we _move GitHub issues that are used to explain process or structure of Marketing Team to the Make Marketing Team Handbook_: https://make.wordpress.org/marketing/handbook

Specifically, I propose that the following two very useful and long-term tickets would be better served as a new "Social Media" page and subsequent "Social Media Campaigns" subpage(s) in the Handbook, as a new section under "[Get Involved](https://make.wordpress.org/marketing/handbook/getting-involved/)

- **WordPress Social Media Amplification ([GB card #7](url))** (authored by @eidolonnight): I suggest moving this to its own "Social Media" page in the Handbook in the Get Involved section

- **[Ongoing] WordPress Photo Directory Social Campaign** - #WPPhotos [GH card #178](https://github.com/WordPress/Marketing-Team/issues/178) (authored by @rmartinezduque): I suggest moving this to a) its own new subpage of the above proposed Social Media page in the Handbook or b) a section of a new "Social Media Campaigns" subpage that provides similar information for the other campaigns listed in the "Process" section of the [Social Media Amplification Card](https://github.com/WordPress/Marketing-Team/issues/7).

As a result of this, the active contribution issues for these campaigns (as referenced in #2 and #3 of the #WPPhotos ticket), can each be their own GitHub issues focused on a specific contributable action, whether that is are ongoing content development, contributor day pushes, or more targeted campaigns (such as creating social content for Learn projects).

**Comment by sereedmedia** [comment]
in my experience, the long explanatory github tickets are confusing because it's not clear how to contribute using that issue and questions/comments are not focused on a single issue. 

we could also discuss putting these longer explanatory texts on GH pages or the GH wiki instead of the handbook, but i personally think we should make the handbook the single-source of truth.

**Comment by rmartinezduque** [comment]
> In my experience, the long explanatory github tickets are confusing because it's not clear how to contribute using that issue, and questions/comments are not focused on a single issue.

I agree. And I think some GitHub issues are confusing not solely because they are long, but because they are not being used as 'tasks' that folks can contribute to. Some are being used to discuss things, share updates, etc.

> we could also discuss putting these longer explanatory texts on GH pages or the GH wiki instead of the handbook, but i personally think we should make the handbook the single-source of truth.

If we want to make the Handbook the single source of truth, I think moving the issues/cards you mentioned to the handbook makes sense. But as I said in Slack, I'd try to make this a consistent process and follow the same path for other issues/cards. For example, a few that come to mind are The Month in WordPress or People of WordPress newsletters. We can also document the process in the Handbook, and then open a card for each edition for contribution.

**Comment by sereedmedia** [comment]
It seems like there is not a lot of feedback on this proposal from folks one way or another, but I am good to move forward with this restructuring with you @rmartinezduque!

Also, I think it would be good to have content in the Handbook restructured prior to the upcoming-at-some-point handbook redesign!

**Comment by sereedmedia** [comment]
We worked on this issues during the Marketing team coffee break over zoom on Thursday March 23. @rmartinezduque @eidolonnight, @mark-andrew and yself @sereedmedia were in attendance. Another team member joined briefly and left.

We talked through the ticket and made a plan to shift the conten, and began moving pages. 

- created an additional page in the Social Media section for campaigns: https://make.wordpress.org/marketing/handbook/social-media-accounts/social-media-campaigns/
added info on the #178 ticket about the #WPPhotos campaign.

Highlighting these campaigns:
[#WordPress](https://make.wordpress.org/marketing/handbook/social-media-accounts/social-media-campaigns/#wordpress) @thetinyl @DanSoschin 
[#LearnWP](https://make.wordpress.org/marketing/handbook/social-media-accounts/social-media-campaigns/#learnwp) @DanSoschin 
[#WPDiversity](https://make.wordpress.org/marketing/handbook/social-media-accounts/social-media-campaigns/#wpdiversity) @santanainniss 
[#WPOpenverse](https://make.wordpress.org/marketing/handbook/social-media-accounts/social-media-campaigns/#wpopenverse) @rmartinezduque 
[#WPPhotos](https://make.wordpress.org/marketing/handbook/social-media-accounts/social-media-campaigns/#wpphotos) 
[#GrowYourStoryWP](https://make.wordpress.org/marketing/handbook/social-media-accounts/social-media-campaigns/#growyourstorywp) @jpantani 
[#WPBriefing](https://make.wordpress.org/marketing/handbook/social-media-accounts/social-media-campaigns/#wpbriefing) @santanainniss 
[#WordCamp](https://make.wordpress.org/marketing/handbook/social-media-accounts/social-media-campaigns/#wordcamp) @bjmcsherry 
[Other updates](https://make.wordpress.org/marketing/handbook/social-media-accounts/social-media-campaigns/#other-updates) @bjmcsherry 

- [ ] Next step is for campaign managers to write summaries of their campaigns modeled after #WPPhotos and add it to the page

- [ ] Possibly bring the Social Media Campaigns page out into its own section so that we can have subpages for each of the campaigns without going three pages deep.

**Comment by rmartinezduque** [comment]
Some quick updates here:

- I updated the [WPPhotos section](https://make.wordpress.org/marketing/handbook/social-media-accounts/social-media-campaigns/#wpphotos) on the new Social Media Campaigns page.
- Added a [summary of the WPOpenverse campaign](https://make.wordpress.org/marketing/handbook/social-media-accounts/social-media-campaigns/#wpopenverse) on the same page.

**Comment by eidolonnight** [comment]
Stale issue.
